pkg/codegen/nodejs/gen.go

```diff
@@ -1571,6 +1571,10 @@ func GeneratePackage(tool string, pkg *schema.Package, extraFiles map[string][]b
 	return files, nil
 }
 
+func PrepareOutDir(outDir string) error {
+	return codegen.CleanDir(outDir, &[]string{"tests"})
```

Comment:
Not all packages have a `tests` directory--the contents of this set should be left to the caller.